### ISSUE TITLE
Floating point exception (core dumped)

### ISSUE THREAD

**Issue by PetrochukM** [pytorch/pytorch#32630]
## 🐛 Bug

Python crashes with "Floating point exception (core dumped)" after running `torch.tensor(0) / torch.tensor(0)`.

## To Reproduce

```
(venv) ubuntu@michael$ python
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.tensor(0) / torch.tensor(1)
tensor(0)
>>> torch.tensor(0) / torch.tensor(0)
^C^C^C^C^CFloating point exception (core dumped)
```

## Expected behavior

Python shouldn't crash.

## Environment

Collecting environment information...
PyTorch version: 1.4.0
Is debug build: No
CUDA used to build PyTorch: 10.1

OS: Ubuntu 18.04.3 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: version 3.13.3

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: 10.0.130
GPU models and configuration:
GPU 0: Tesla T4
GPU 1: Tesla T4
GPU 2: Tesla T4
GPU 3: Tesla T4

Nvidia driver version: 418.87.01
cuDNN version: /usr/local/cuda-10.1/targets/x86_64-linux/lib/libcudnn.so.7.6.2

Versions of relevant libraries:
[pip3] numpy==1.18.0
[pip3] pytorch-nlp==0.5.0
[pip3] torch==1.4.0
[conda] Could not collect

**Comment by ngimel** [comment]
Related issue and discussion #327 (don't be fooled by 3-digit number, it's active as of now). @PetrochukM please check if this issue can be closed in favor of #327

**Comment by PetrochukM** [comment]
Haha wow, that's from 2016!

Closing!
